draft-ietf-httpbis-cache-latest.xml

```diff
@@ -331,7 +331,10 @@
   <li><t>The request method is understood by the cache, and</t></li>
   <li><t>the response status code is final (see <xref
   target="associating.response.to.request"/>), and</t></li>
-  <li><t>the response status code is understood by the cache, and</t></li>
+  <li><t>the response status code is understood by the cache, if it is 206, 
```

Comment:
It took me a moment to parse this. Why do 206 and 304 show up here now? Is this adressing the same issue? Maybe a separate paragraph about what status codes a cache MUST understand would help?

Reply:
Yes, this was the interesting part. Partial content and validation responses are two parts of HTTP where a cache has to be aware of the semantics of the response to store it; if they aren't, bad things can happen -- and those responses often do carry caching metadata. So calling them out explicitly seemed wise.

This is definitely part of this issue; it's assumed in the current text, but now needs to be explicit because of the change. 

Would it clarify to split these out into a separate bullet point, with a small amount of explanatory text?

Reply:
yes.